store/postgres/src/writable.rs

```diff
@@ -438,10 +439,16 @@ enum Request {
         block_ptr: BlockPtr,
         firehose_cursor: FirehoseCursor,
     },
+    Stop,
+}
+
+enum ExecResult {
+    Continue,
+    Stop,
```

Comment:
Just a suggestion, so feel free to ignore this comment.

This enum somewhat similar to the [ControlFlow](https://doc.rust-lang.org/std/ops/enum.ControlFlow.html) from `std`. 

While we don't need to return any data out of this, maybe we could redefine it as
```rust
type ExecResult = ControlFlow<()>
```

I know this won't change anything, but it would make room if we want to return data from those contexts in the future.